gui/src/2D/controls/control.ts

```diff
@@ -1122,6 +1122,19 @@ export class Control {
 
         this._isEnabled = value;
         this._markAsDirty();
+        // if this control or any of it's descendants are under a pointer, we need to fire a pointerOut event
+        const recursivelyFirePointerOut = (control: Control) => {
+            for (const pointer in control.host._lastControlOver) {
+                if (control === this.host._lastControlOver[pointer]) {
+                    control._onPointerOut(control, null, true);
+                    delete control.host._lastControlOver[pointer];
+                }
+            }
+            if (control instanceof Container) {
```

Comment:
would it be enough to check if `control.children` exists instead of using instanceof?